packages/manager/modules/nutanix/src/dashboard/general-info/edit-display-name/module.js

```diff
@@ -0,0 +1,26 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'angular-translate';
+import '@ovh-ux/ng-translate-async-loader';
+import '@ovh-ux/ui-kit';
+
+import { editServiceName } from '@ovh-ux/manager-components';
+import component from './component';
+import routing from './routing';
+
+const moduleName = 'ovhManagerNutanixGeneralInfoEditDisplayName';
+
+angular
+  .module(moduleName, [
+    'oui',
+    'pascalprecht.translate',
+    'ngTranslateAsyncLoader',
+    'ngUiRouterBreadcrumb',
+    'ui.router',
+    editServiceName,
+  ])
+  .config(routing)
+  .component('nutanixGeneralInfoEditDisplayName', component)
```

Comment:
We have to prefix it with 'Component':

```suggestion
  .component('nutanixGeneralInfoEditDisplayNameComponent', component)
```